videomaker/inc/patterns/header-names-list-and-grid.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Header, Names List and Grid
+ *
+ * @package Videomaker
+ */
+
+return array(
+	'title'      => __( 'Header, Names List and Grid', 'videomaker' ),
+	'categories' => array( 'videomaker' ),
+	'blockTypes' => array( 'core/template-part/header' ),
+	'content'    => '<!-- wp:spacer -->
+	<div style="height:100px" aria-hidden="true" class="wp-block-spacer"></div>
+	<!-- /wp:spacer -->
+
+	<!-- wp:columns {"verticalAlignment":"top","align":"wide"} -->
+	<div class="wp-block-columns alignwide are-vertically-aligned-top"><!-- wp:column {"verticalAlignment":"top","width":"46.67%"} -->
+	<div class="wp-block-column is-vertically-aligned-top" style="flex-basis:46.67%"><!-- wp:site-title /-->
+
+	<!-- wp:heading {"level":3,"style":{"typography":{"fontWeight":"300"}}} -->
+	<h3 style="font-weight:300">Augusta Smith<br>Dan Andrews<br>Syd Wyatt<br>Philip Mizrahi<br>Tony Sánchez<br>Romana Arak</h3>
+	<!-- /wp:heading --></div>
+	<!-- /wp:column -->
+
+	<!-- wp:column {"width":"13.33%"} -->
+	<div class="wp-block-column" style="flex-basis:13.33%"><!-- wp:spacer -->
+	<div style="height:100px" aria-hidden="true" class="wp-block-spacer"></div>
+	<!-- /wp:spacer --></div>
+	<!-- /wp:column -->
+
+	<!-- wp:column {"width":"60%"} -->
+	<div class="wp-block-column" style="flex-basis:60%">
+
+	<!-- wp:video {"src":"' . get_stylesheet_directory_uri() . '/assets/videos/boats.mp4","videoPressClassNames":"wp-block-embed is-type-video is-provider-videopress"} -->
+	<figure class="wp-block-video wp-block-embed is-type-video is-provider-videopress"><video src="' . get_stylesheet_directory_uri() . '/assets/videos/boats.mp4"></video></figure>
+	<!-- /wp:video -->
+
+	<!-- wp:heading {"level":3,"style":{"typography":{"fontWeight":"700"}},"fontSize":"normal"} -->
+	<h3 class="has-normal-font-size" style="font-weight:700">' . esc_html__( 'Boats', 'videomaker' ) . '</h3>
```

Comment:
@beafialho also wondering if these headers should match the theme?

Reply:
You're referring to the people's names, right? This navigation would have different styles from the header on top, so I think we can probably leave this pattern out for now.

cc: @kjellr